sagemaker/launch/launch_partition.py

```diff
@@ -90,7 +91,7 @@ def get_partition_parser():
 
     partition_args = parser.add_argument_group("GraphStorm Partition Arguments")
 
-    partition_args.add_argument("--output-data-s3", type=str,
+    partition_args.add_argument("--output-data-s3", "--output-graph-s3", type=str,
```

Comment:
Why add `--output-graph-s3`?

Reply:
Trying to make launch_partition interface consistent with GConstruct launcher. Users ideally won't shouldn't need to remember a different argument name for the same purpose for each launcher.